systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -218,11 +215,17 @@ void testZookeeperScaleUpScaleDown() {
         replaceKafkaResource(CLUSTER_NAME, k -> k.getSpec().getZookeeper().setReplicas(initialZkReplicas));
 
         for (String name : newZkPodNames) {
-            kubeClient.waitForResourceDeletion("po", name);
+            KUBE_CLIENT.waitForResourceDeletion("po", name);
```

Comment:
"po" -> "pod" here?

Reply:
it's the same from oc client POV, but I can but there `pod`. It make sense to me.